cmds/cmd_moderation.py

```diff
@@ -498,6 +516,50 @@ async def grab_guild_message(message, args, client, **kwargs):
         await message.channel.send("There were files attached but they exceeded the guild filesize limit", embed=message_embed)
 
 
+class purger:
+    def __init__(self, user_id: int):
+        self.user_id = user_id
+
+    def check(self, message: discord.Message) -> bool:
+        return bool(message.author.id == self.user_id)
+
+
+async def purge_cli(message: discord.Message, args: List[str], client: discord.Client, **kwargs: Any) -> Any:
+
+    if args:
+        try:
+            limit = int(args[0])
+        except ValueError:
+            await message.channel.send("ERROR: Limit is not valid int")
+            return 1
+    else:
+        await message.channel.send("ERROR: No limit specified")
+        return 1
+
+    if limit > 100:
+        await message.channel.send("ERROR: Cannot purge more than 100 messages")
+        return 1
+
```

Comment:
what if the users put in 0 or negative numbers?

Reply:
oh dear